libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx

```diff
@@ -291,7 +321,7 @@ export function UniversalDappUI(props: UdappProps) {
           { props.isSavedContract && props.instance.filePath ? (
             <div className="d-flex" data-id="instanceContractFilePath">
               <label>
-                <b><FormattedMessage id="udapp.filePath" />:</b> {props.instance.filePath}
+                <span style={{float: 'left'}}><b><FormattedMessage id="udapp.filePath" />:</b></span> &nbsp;{props.instance.filePath}
```

Comment:
Using a class here for float will be better and also margin/padding in place of `'&nbsp;'`

Reply:
I will remove that as of now, as in next PR we can remove this field